test/e2e/network/service.go

```diff
@@ -1217,6 +1217,32 @@ var _ = SIGDescribe("Services", func() {
 		framework.ExpectNoError(err)
 	})
 
+	ginkgo.It("should be able to create a functioning ClusterIP service with ExternalIPs", func() {
```

Comment:
So as mentioned in https://github.com/kubernetes/kubernetes/issues/95785#issuecomment-715312126 there are two different codepaths for external IPs, depending on whether the external IP is assigned to one of the node's interfaces or not, and this test only checks the "not" case. So it would be good to clarify that (and also, to leave #95785 open because this doesn't fully resolve it).

Something like `It("should be possible to connect to a service via ExternalIP when the external IP is not assigned to a node")`

Reply:
I'd rather find a way to satisfy the other code path, do you have any ideas on how we should go about adding / getting an external IP to a node?

Reply:
that is very disruptive since you have to add IPs to the node, I rather prefer to tackle that in another PR and have this merged to have a minimum of coverage

Reply:
@aojea sounds good.